src/scss/components/navigation/_index.scss

```diff
@@ -56,17 +59,28 @@
     text-decoration: none;
 
     &.current {
-      background-color: $gray-900;
-      color: $gray-0;
+      background-color: $gray-200;
+      border-left: 0.125rem solid $purple-500;
+      color: $purple-500;
+      margin-left: 1.25rem;
+      padding-left: 1.125rem;
     }
 
     &:hover {
-      background-color: $gray-800;
-      color: $gray-0;
+      background-color: $gray-200;
+      color: $purple-500;
     }
   }
 
   hx-reveal {
+    a.active {
```

Comment:
You should be able to add styles based on the `:active` pseudo-class.  Try using `a:active` here.

Reply:
Changes have been done, Please review